lib/src/utils.dart

```diff
@@ -69,7 +69,7 @@ String suggestLibraryName(AssetId source) {
   parts[parts.length - 1] = parts.last.split('.').first;
   // [lib/test] --> [test]
   if (parts.first == 'lib') {
-    parts = parts.skip(1);
+    parts.removeAt(0);
```

Comment:
nit: the skip is probably actually faster in this scenario

Reply:
Yes, but then we're trying Iterables like Lists...

Reply:
ah ya ok probably not worth going through the type shenanigans just for a minor optimization here

Reply:
t'was my thought